frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleBusinessManagerPage/AppleBusinessManagerPage.tsx

```diff
@@ -73,6 +76,7 @@ const AppleBusinessManagerPage = ({ router }: { router: InjectedRouter }) => {
   const [showRenewModal, setShowRenewModal] = useState(false);
   const [showDeleteModal, setShowDeleteModal] = useState(false);
   const [showAddAbmModal, setShowAddAbmModal] = useState(false);
+  const [showEditTeamModal, setShowEditTeamModal] = useState(false);
```

Comment:
I don't know if you want to rename this showEditTeamsModal (teams plural) or not